.github/workflows/deploy.yml

```diff
@@ -333,119 +284,6 @@ jobs:
           name: ${{ fromJSON(env.SHOULD_DEPLOY_PRODUCTION) && 'desktop-build-artifact' || 'desktop-staging-build-artifact' }}
           path: ./desktop-build/NewExpensify.dmg
 
-  iOS:
```

Comment:
we should rename `android_hybrid` to just `android` and `ios_hybrid` to just `iOS`